### DISCUSSION TITLE
contest modul - flexible free label of a local contest

### DISCUSSION THREAD

**Discussion by DK5EW** [wavelog/wavelog#881] [Ideas]
I see in the contest module that there are many pre-defined contests.
I understand that not all available contests can be in the list.

For example, if I want to take part in a local contest, of course I can't find anything, but the flexibility of the predefined input fields would allow me to run this local contest.
![free contest label](https://github.com/user-attachments/assets/24d196b1-1265-4669-b5ed-7087ba9f2d45)

Only the name cannot be changed in the contest pull-down menu.
How about adding a field here that allows a free name for the contest?

**Comment by HB9HIL** [comment]
You can add contests in the Admin Menu. If you are no admin just tell the admin which contest should be added to the list :)
